app-imaging/glslang/spec

```diff
@@ -1,4 +1,4 @@
-VER=11.11.0
+VER=13.1.1
 SRCS="tbl::https://github.com/KhronosGroup/glslang/archive/$VER.tar.gz"
```

Comment:
Please try not to use automatically packaged tarballs, use Git tags instead.

Reply:
Fixed.